htlcswitch/link.go

```diff
@@ -3728,6 +3728,11 @@ func (l *channelLink) processRemoteSettleFails(fwdPkg *channeldb.FwdPkg) {
 //
 //nolint:funlen
 func (l *channelLink) processRemoteAdds(fwdPkg *channeldb.FwdPkg) {
+	// Exit early if there are no adds to process.
+	if len(fwdPkg.Adds) == 0 {
+		return
```

Comment:
Let's add a log here since it's unexpected to have empty `Adds` here

Reply:
It is not unexpeted tho, every HTLC will eventually be resolved and then the fwd has only Fail/Settles no ADDs, I can add a trace if you want ?

Reply:
yeah right - we sort just move the empty check into the methods, so they are always called.